matchzoo/preprocessors.py

```diff
@@ -0,0 +1,133 @@
+"""Matchzoo toolkit for text pre-processing."""
+
+import re
+import nltk
+import jieba
```

Comment:
I switch to `thulac` from `jieba`, but that's not a big deal, just keep in mind that maybe multiple backends will be supported in the future.

Reply:
leave it for now